Template/Coordinators/AppCoordinator.swift

```diff
@@ -2,54 +2,37 @@
 //  AppCoordinator.swift
 //  Template
 //
-//  Created by Rameez Khan on 16/10/21.
+//  Created by Apple on 03/03/22.
 //
 
+import Foundation
 import UIKit
+
+
```

Comment:
run swiftformat on this file
`swiftformat Template/Coordinators/AppCoordinator.swift`